util/reggen/ip_block.py

```diff
@@ -19,6 +20,45 @@
 from reggen.signal import Signal
 from reggen.countermeasure import CounterMeasure
 
+# Known comportable IP names and associated CIP_IDs.
```

Comment:
Nit: known **unique** comportable IP names and associated CIP_IDs.

Reply:
Will do!

Reply:
Done.